keras/utils/layer_utils.py

```diff
@@ -21,6 +21,7 @@
 
 from keras.utils import io_utils
 from keras.utils import tf_inspect
+from keras.utils.vis_utils import get_layer_index_bound_by_layer_name
```

Comment:
Please import the module only, rather than import the function directly.

Reply:
This import was breaking many tests for other functionalities because they didn't find `vis_utils` in `keras.utils` so i decided to move the `get_layer_index_bound_by_layer_name` function to 'layer_utils'. This solved the import error but i had to add some change to the `keras.utils.vis_utils` module.